usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -227,6 +227,10 @@ LIBS+=(
 )
 
 COPY_AS_IS+=( /dev /etc/inputr[c] /etc/protocols /etc/services /etc/rpc /etc/termcap /etc/terminfo /lib*/terminfo /usr/share/terminfo /etc/netconfig /etc/mke2fs.conf /etc/*-release /etc/localtime /etc/magic /usr/share/misc/magic /etc/dracut.conf /etc/dracut.conf.d /usr/lib/dracut /sbin/modprobe.ksplice-orig /etc/sysctl.conf /etc/sysctl.d /etc/e2fsck.conf )
+
+# Needed by vi on Fedora and derived distributions
+COPY_AS_IS+=( /usr/libexec/vi )
```

Comment:
@pcahyna 
please add to this comment a line with a URL
to this pull request so one can see the reason behind
(e.g. for me as non Fedora user the reason is not obvious
 I could imagine some reason but I won't know for sure)
or tell the reason directly in the comment like
```
# Needed by vi on Fedora and derived distributions
# where vi is a shell script that executes /usr/libexec/vi
# see https://github.com/rear/rear/pull/2822
```